scripts/setup-jest-integration-tests.ts

```diff
@@ -30,6 +30,13 @@ afterEach(() => {
 	}
 });
 
+beforeEach(async () => {
```

Comment:
Is this necessary? When we run integration tests with `npm run test:it`, we run [`prisma migrate reset`](https://www.prisma.io/docs/concepts/components/prisma-migrate/migrate-development-production#reset-the-development-database), which should delete any data from tests.

If we want to do this in code so that it's done on every integration test run (not just when we use `npm run test:it`), we could execute the reset command as a child process. Or maybe look at the workarounds suggested in https://github.com/prisma/prisma/issues/5230

Reply:
Just realised this is executing after each test so it's not doing the same thing as `npm run test:it`. I'm happy to merge this as is.

Reply:
Justin, if I understand correctly, our `npm run test:it` currently takes care of cleaning data and applying the migrations (which is the primary task!) before running tests.

However, it can be also beneficial to clean data before _each_ test to prevent stale data (e.g., from a crashed test) to impact other tests. We already do this for some integration tests: please, see [here](https://github.com/atlassian-labs/figma-for-jira/blob/add6f10ae30a58dc14987fa9c3026abb52beb988/src/web/routes/entities/integration.test.ts#L272) for an example.

The PR just makes this automatic and more reliable. Please, pay attention that we do the same thing in other projects.